x/bridge/observer/osmosis/client.go

```diff
@@ -50,18 +55,21 @@ func NewClient(
 	return NewClientWithConnection(chainId, grpcConn, keyring), nil
 }
 
+// NewClientWithConnection returns new instance of `Client`
+// using provided gRPC connection
 func NewClientWithConnection(
 	chainId string,
 	conn *grpc.ClientConn,
 	keyring keyring.Keyring,
 ) Client {
 	return Client{
-		chainId:   chainId,
-		keyring:   keyring,
-		grpcConn:  conn,
-		txConfig:  app.GetEncodingConfig().TxConfig,
-		txClient:  tx.NewServiceClient(conn),
-		accClient: authtypes.NewQueryClient(conn),
+		chainId:      chainId,
+		keyring:      keyring,
+		grpcConn:     conn,
+		txConfig:     app.GetEncodingConfig().TxConfig,
```

Comment:
This line will lead to a cyclic import with the `app` package during wiring. Let's just pass this value as an argument